modules/ROOT/pages/coverage-studio-concept.adoc

```diff
@@ -6,51 +6,69 @@ endif::[]
 You can run a coverage report from Anypoint Studio.
 
 The following configurations apply only when you execute your MUnit tests in Studio. They do not apply when running tests from Maven in a CI/CD environment. +
-To run coverage reports from Maven see xref:coverage-maven-concept.adoc[Using Coverage in Maven].
+To run coverage reports from Maven see xref:coverage-maven-concept.adoc[].
 
 == Enable Coverage Reports in Anypoint Studio
 
-The Coverage button in the image bellow allows you to see which flows, and which percentage of event processors in those
-flows were covered by the test:
+The Coverage button enables you to see which flows, and which percentage of event processors in those flows the test covers:
 
-image::using-munit-coverage-in-anypoint-studio.png[]
+image::using-munit-coverage-in-anypoint-studio.png["MUnit Coverage tab highlighted in the MUnit section."]
 
-The Overall Coverage value represents the percentage of the Mule application event processors that have been executed by the MUnit test.
+The Overall Coverage value represents the percentage of the Mule application event processors that the MUnit test executes.
 
-You can click the Generate Report button to get the report detailing the flows and event processors contained by the Mule configuration files that were included in the MUnit test.
+You can click on *Generate Report* to get the report that details the flows and event processors in the Mule configuration files that are included in the MUnit test.
 
-image::munit-coverage-report-in-anypoint-studio.png[]
+image::munit-coverage-report-in-anypoint-studio.png["Files included in the report are highlighted."]
 
 [%header,cols="20a,80a"]
 |===
 | Column      | Description
-| *Resource*    | The name of the Mule Configuration file.
-| *Containers* | Number of Flows inside the Mule configuration file.
-| *Weight*      | This field represents how much of your application is contained inside a configuration file. +
+| *Resource*    | The name of the Mule Configuration file
+| *Containers* | Number of flows inside the Mule configuration file
```

Comment:
```suggestion
| *Containers* | Number of flows in the Mule configuration file
```

Reply:
(I just looked up in vs inside because I always default to in.)

Reply:
For me, "inside" seems to be an emphasis on "in" hahahahah

Reply:
I asked Diane and she didn't know the difference. I always use "in" because it's shorter and costs less to translate :>

Reply:
Good to know! Thanks!